### DISCUSSION TITLE
Help! I need your dreambooth tips/lessons learned in training custom models of a person?

### DISCUSSION THREAD

**Discussion by juan9999** [TheLastBen/fast-stable-diffusion#1009] [Q&A]
Does anyone have a summary of best practices/lessons learned training with dreambooth on people?

I've been having issues producing a good custom model using dreambooth.

Here is what I've tried
- using SD 2.1 768
- using ~40 pictures (renamed to a non-existing work) resized to 756X756  (no other people, different expressions, backgrounds, angles, 20X close 6X side, 10 chest up, 6 full body)
- setting UNet training to 1000
- 400 text encoder training steps (I'm not sure what this does)
- I skip Concept Images
- I don't "tag" photos which I've heard some people do

I then tried:
- 26 pictures
- setting UNet training 101*[number of pictures] (2626)
But first model seems to produce better results

I haven't used tags, 

Most training was done with Tesla T4

**Comment by TheLastBen** [comment]
The V2 is harder to train than the V1.5, right now I'm working on finding the optimal settings to train on faces, hopefully soon I will find the right combination of steps and leaning rate.
tip: use only 15-20 instance images for better training.

**Reply by gilroff** [reply]
> The V2 is harder to train than the V1.5, right now I'm working on finding the optimal settings to train on faces, hopefully soon I will find the right combination of steps and leaning rate. tip: use only 15-20 instance images for better training.

Won't more instance images in different settings but still keeping only the same face, improve the training? I am still new to this. I dont think i really understand what concept images exactly do. can you explain how instance image training, text encoder and concept image training actually work in a simplified way? Or is there a documentation like that I can check out? Please give me link if there is. Btw thanks for your awesome work <3

**Reply by TheLastBen** [reply]
to many instance images will split the number of steps per image and will result in poor quality, 10-20 instance images is enough to capture the face or the style, then the concept images will help the text_encoder widen it's range of output, concept images can be as many as 200-300 to help with diverse concepts.

**Reply by gilroff** [reply]
ok thanks i get you

**Comment by juan9999** [comment]
Would you suggest using 1.5 for now? I got used to negative prompts and 764 image sizes lol! 

Can I use a 1.5 model in 2.1? How "big" can I go on an image if model is trained at 512?

**Reply by TheLastBen** [reply]
with v1.5 512x512 you can go as big as 1280x1280 using the highres.fix feature, here is an example on a model I trained on 11 images : 

![00299-2122058592-group building a house with ropes  in  ttdddd , moss, moist, ultra high quality, high quality graphical details,  cinematic, vol](https://user-images.githubusercontent.com/110583491/207439480-37284c13-1aad-41df-a24b-1ed02b1c606b.png)
![00029-2341924608-ttdddd ,a cute hobbit casting spells as the Sorcerer Supreme in Medieval armor, HD,ultra realistic, ultra detailed, realistic, 3](https://user-images.githubusercontent.com/110583491/207439511-0744afe9-a1e1-41f4-8ebb-4b922848389c.png)
![00321-2941767663-ttdddd ,a clown riding a unicycle, HD,ultra realistic, ultra detailed, realistic, 3d render, cinematic, photorealistic](https://user-images.githubusercontent.com/110583491/207439836-7365ed06-f021-4976-b600-e591c69a5503.png)

**Reply by leonnn1** [reply]
Dude can you share this model, it's awesome!!

**Reply by TheLastBen** [reply]
https://huggingface.co/TheLastBen
There are two versions, for v1.5 and v2.1

**Reply by leonnn1** [reply]
> https://huggingface.co/TheLastBen There are two versions, for v1.5 and v2.1

Thank you a lot!

**Reply by leonnn1** [reply]
> https://huggingface.co/TheLastBen There are two versions, for v1.5 and v2.1

I finally made a model using 2.1, took me around 4 hours of training but it's the first pixel art model I do that works, pretty amazing, thank you!
![image](https://user-images.githubusercontent.com/85142472/208029507-b659d1b5-54aa-4a44-b200-addf1b3a9aa8.png)
![image](https://user-images.githubusercontent.com/85142472/208029525-0848542d-a65d-4449-a4c3-ac95efc5da22.png)
![image](https://user-images.githubusercontent.com/85142472/208029547-31ce906c-162d-4bea-903d-12c37ac04f84.png)
![image](https://user-images.githubusercontent.com/85142472/208029575-29474f4f-945b-4439-b504-bf20e6e22e32.png)

**Comment by juan9999** [comment]
Wow great work! What was the subject of the training?

**Reply by TheLastBen** [reply]
It was pictures of frogs in a swamps
